cmd/kubelet/app/options/options.go

```diff
@@ -80,6 +80,10 @@ type KubeletFlags struct {
 
 	// Container-runtime-specific options.
 	ContainerRuntimeOptions
+
+	// Experimental flag that enables shared/slave propagation for mounts in
+	// pods. It may be redesigned or even removed in future releases!
+	MountPropagation bool
```

Comment:
renamed to `--experimental-mount-propagation` (we agreed on `--test-*` on the meeting though...)